build.gradle

```diff
@@ -0,0 +1,110 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This is a general purpose Gradle build.
+ * Learn how to create Gradle builds at https://guides.gradle.org/creating-new-gradle-builds
+ */
+
+plugins {
+	id 'java'	
```

Comment:
May want to use the [java-library](https://docs.gradle.org/current/userguide/java_library_plugin.html) plugin instead